app/controllers/places_controller.rb

```diff
@@ -6,6 +6,10 @@ def show
     @vote = @place.votes.new(uuid: uuid)
   end
 
+  def index
```

Comment:
I'm not sure I agree that this is an index. It doesn't give me a list of all of the places I can search or explore. It gives me a subset. I'd suggest calling the action `leaderboard` instead.